app/views/app/show.html.erb

```diff
@@ -30,4 +30,11 @@
 
     <%= render "shared/screenshots", post: @post if @post.ordered_images.any? %>
   <% end %>
+
+  <%= govuk_one_third do %>
+    <h2 class="govuk-heading-s">
+      Related links
+    </h2>
+    <%= GovukMarkdown.render(@post.related_links).html_safe %>
+  <% end %>
```

Comment:
@fofr do you think this needs anything more for now? Borders etc?

Reply:
Will steal from https://github.com/x-govuk/govuk-prototype-components/blob/main/x-govuk/components/related-navigation/_related-navigation.scss